src/main/java/com/exadel/discount/entity/Coupon.java

```diff
@@ -0,0 +1,39 @@
+package com.exadel.discount.entity;
+
+import com.exadel.discount.dto.coupon.BaseCouponDto;
+import com.exadel.discount.dto.coupon.CouponDto;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Data
+@Entity
+@NoArgsConstructor
+@Table(name = "coupons")
+
+public class Coupon {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private UUID id;
+    private LocalDateTime date;
```

Comment:
I suppose it's a good practice to add @Column annotation everywhere.

Reply:
If column name is different of field's one. Here it will be just a redundant chars.

Reply:
![image](https://user-images.githubusercontent.com/54329787/122365578-72d1dc00-cf63-11eb-82be-ba5b27285f27.png)
Sergey left this comment under Nikolay's PR. Maybe it's just a good practice